bower.json

```diff
@@ -12,7 +12,6 @@
     "iron-resizable-behavior": "polymerelements/iron-resizable-behavior#^1.0.0"
   },
   "devDependencies": {
-    "carbon-elements": "polymerelements/carbon-elements#gh-pages",
```

Comment:
I would recommend switching to Frankie's suggestion. `gh-pages` is the default branch on that repo and just specifying the raw git URL is sufficient to pull it down.